test/jdk/java/awt/Mixing/AWT_Mixing/SimpleOverlappingTestBase.java

```diff
@@ -141,20 +158,29 @@ protected boolean performTest() {
            tests fail starting after failing mixing tests but always pass alone.
          */
         JFrame ancestor = (JFrame)(testedComponent.getTopLevelAncestor());
-        if( ancestor != null ) {
-            Point ancestorLoc = ancestor.getLocationOnScreen();
-            ancestorLoc.translate(isOel7orLater() ? 5 :
-                                             ancestor.getWidth() / 2 - 15, 2);
-            robot.mouseMove(ancestorLoc.x, ancestorLoc.y);
-            Util.waitForIdle(robot);
-            robot.mousePress(InputEvent.BUTTON1_MASK);
-            robot.delay(50);
-            robot.mouseRelease(InputEvent.BUTTON1_MASK);
-            Util.waitForIdle(robot);
+
+        if (ancestor != null) {
+            final CountDownLatch latch = new CountDownLatch(1);
+            ancestor.addFocusListener(new FocusAdapter() {
+                @Override public void focusGained(FocusEvent e) {
+                    latch.countDown();
+                }
+            });
+            ancestor.requestFocus();
+            try {
+                boolean await = latch.await(1L, TimeUnit.SECONDS);
+                if (!await) {
```

Comment:
```suggestion
                if (!latch.await(1L, TimeUnit.SECONDS)) {
```
I don't see any value in saving the result of `await` in a variable; the call itself isn't long enough; calling `latch.await` in the condition is a common pattern.